library/Zend/Cache/Storage/Adapter/MemcacheOptions.php

```diff
@@ -44,6 +45,58 @@ class MemcacheOptions extends AdapterOptions
     protected $compression = false;
 
     /**
+     * {inheritdoc}
+     */
+    public function setFromArray($options)
+    {
+        if ($options instanceof AbstractOptions) {
+            $options = $options->toArray();
+        }
+
+        if (!is_array($options) && !$options instanceof Traversable) {
+            throw new Exception\InvalidArgumentException(
+                sprintf(
+                    'Parameter provided to %s must be an %s, %s or %s',
+                    __METHOD__,
+                    'array',
+                    'Traversable',
+                    'Zend\Stdlib\AbstractOptions'
+                )
+            );
+        }
+
+        if ($options instanceof Traversable) {
+            $options = iterator_to_array($options);
+        }
+
+        $options  = array_change_key_case($options, CASE_LOWER);
+        $prioOpts = array();
```

Comment:
What does `$prioOpts` stand for? `prior options`? or `priority options`? or...? Please be explicit. :)